cmd/execution_builder.go

```diff
@@ -223,6 +223,11 @@ func (builder *ExecutionNodeBuilder) LoadComponentsAndModules() {
 		// I prefer to use dummy component now and keep the bootstrapping steps properly separated,
 		// so it will be easier to follow and refactor later
 		Component("execution state", exeNode.LoadExecutionState).
+		// Load the admin tool when chunk data packs db are initialized in execution state
+		AdminCommand("create-chunk-data-packs-checkpoint", func(config *NodeConfig) commands.AdminCommand {
+			// checkpoints will be created under "/data/chunk_data_packs_checkpoints"
+			return storageCommands.NewChunksCheckpointCommand("/data/chunk_data_packs_checkpoints", exeNode.chunkDataPackDB)
```

Comment:
The admin tool can create checkpoint, which allows us to query pebble db without stopping the node.